integration_tests/sdk/delete_workflow_test.py

```diff
@@ -94,6 +95,8 @@ def test_delete_workflow_saved_objects(client):
 
         client.delete_flow(flow_id, saved_objects_to_delete=tables, force=True)
 
+        flow_ids_to_delete.remove(flow_id)
+
```

Comment:
Can we also apply the same changes to `test_delete_workflow_saved_objects()`? The polling and use of .get(). The polling in particular seems useful as a helper in this file.

Reply:
Also, if I'm understanding this test correctly, even if the `delete_flow()` partially succeeds by deleting all the metadata, you can still call it again to delete tables?

Reply:
For this specific test I check the flow with object(s) saved with update_mode=APPEND can only be deleted if in force mode.